packages/trader/src/AppV2/Components/ContractDetailsFooter/contract-details-footer.tsx

```diff
@@ -36,11 +35,13 @@ const ContractDetailsFooter = observer(({ contract_info }: ContractInfoProps) =>
     const is_multiplier = isMultiplierContract(contract_type);
 
     const cardLabels = getCardLabelsV2();
-    const bidDetails = !is_valid_to_cancel ? `@${bid_price} ${currency}` : '';
+    const formatted_bid_price = FormatUtils.formatMoney(bid_price || 0, {
+        currency: currency as 'USD', // currency types mismatched between utils and shared
```

Comment:
we have currencies other than USD though ?

Reply:
sure, it's just a type here in order to not use `any` :)